tools/ci_build/github/apple/get_simulator_device_info.py

```diff
@@ -137,13 +143,19 @@ def runtime_id_and_device_pair_key(runtime_id_and_device_pair):
 
 
 def main():
+    requested_runtime_version_environment_variable_name = "ORT_GET_SIMULATOR_DEVICE_INFO_REQUESTED_RUNTIME_VERSION"
+
     parser = argparse.ArgumentParser(description="Gets simulator info from Xcode and prints it in JSON format.")
-    parser.add_argument("--max-runtime-version", help="The maximum runtime version to allow.")
```

Comment:
nit: would using ArgumentDefaultsHelpFormatter print the default value for the requested runtime version? IIRC it can be inconsistent if the default is dynamically set.

https://docs.python.org/3/library/argparse.html#argparse.ArgumentDefaultsHelpFormatter

Reply:
the latest available version would need to be queried. I just added some more help text to indicate what an unset value means.